src/App.vue

```diff
@@ -295,8 +295,8 @@ export default {
 
         // set the user for the chatwoot instance
         window.$chatwoot.setUser(this.user.id, {
-          email: this.user.email,
-          name: this.user.first_name + this.user.last_name,
+          email: this.user.email || "no_email",
```

Comment:
this will probably give error that `undefined has no attribute email`

Reply:
No this piece of code runs only when `isAuthenticated` is true